doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+
+::
+
+  slicing and indexing
+
+  np.vstack()
+  np.hstack()
+  np.hsplit()
+  
+  .view()
+  .copy()
+
+-----
+
+You can easily use create a new array from a section of an existing array. 
+
+Let's say you have this array:
+
+::
+
+  array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10])
+
+You can create a new array from a section of your array any time by specifying where you want to slice your array.
+
+**Input:**
+
+::
+
+  arr1 = np.array[3:8]
+  arr1
+
+**Output:**
+
+::
+
+  array([4, 5, 6, 7, 8])
+
+Here, you grabbed a section of your array from index position 3 through index position 8.
+
+You can also stack two existing arrays, both vertically and horizontally. Let's say you have two arrays: 
+
+**a_1**:
+
+::
+
+  array([[1, 1],
+       [2, 2]])
+
+and **a_2**:
+
+::
+
+  array([[3, 3],
+       [4, 4]])
+
+You can stack them vertically with ``vstack``:
+
+**Input:**
+
+::
+
+  np.vstack((a_1, a_2))
+
+**Output:**
+
+::
+
+  array([[1, 1],
+       [2, 2],
+       [3, 3],
+       [4, 4]])
+
+Or stack them horizontally with ``hstack``:
+
+**Input:**
+
+::
+
+  np.hstack((a_1, a_2))
+
+**Output:**
+
+::
+
+  array([[1, 1, 3, 3],
+       [2, 2, 4, 4]])
+
+You can split an array into several smaller arrays using ``hsplit``. You can specify either the number of equally shaped arrays to return or the columns *after* which the division should occur.
+
+Let's say you have this array:
+
+::
+
+  array([[ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12],
+       [13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24]])
+
+If you wanted to split this array into three equally shaped arrays, you would run:
+
+**Input:**
+
+::
+
+  np.hsplit(array,3)
+
+**Output:**
+
+::
+
+  [array([[ 1,  2,  3,  4],
+        [13, 14, 15, 16]]), array([[ 5,  6,  7,  8],
+        [17, 18, 19, 20]]), array([[ 9, 10, 11, 12],
+        [21, 22, 23, 24]])]
+
+If you wanted to split your array after the third and fourth column, you'd run:
+
+**Input:**
+
+::
+
+  np.hsplit(array,(3,4))
+
+**Output:**
+
+::
+
+  [array([[ 1,  2,  3],
+        [13, 14, 15]]), array([[ 4],
+        [16]]), array([[ 5,  6,  7,  8,  9, 10, 11, 12],
+        [17, 18, 19, 20, 21, 22, 23, 24]])]
+
+`Learn more about stacking and splitting arrays here <https://docs.scipy.org/doc/numpy-1.17.0/user/quickstart.html#stacking-together-different-arrays>`_.
+
+You can use the ``view`` method to create a new array object that looks at the same data as the original array (a *shallow copy*)
+
+Let's say you create this array:
+
+::
+
+  a = np.array([[1 , 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+
+You can create a new array object that looks at the same data using:
+
+**Input:**
+
+::
+
+  b = a.view()
+
+Using the ``copy`` method will make a complete copy of the array and its data (a *deep copy*). To use this on your array, you could run:
+
+**Input:**
+
+::
+
+  c = a.copy()
+ 
+`Learn more about copies and views here <https://docs.scipy.org/doc/numpy-1.17.0/user/quickstart.html#copies-and-views>`_.
```

Comment:
Better to do a replace in the whole doc from https://docs.scipy.org/doc/numpy-1.17.0 to https://www.numpy.org/devdocs